backend/app/crud.py

```diff
@@ -31,26 +31,26 @@ def get_users(db: Session, skip: int = 0, limit: int = 100) -> List[models.User]
     return db.query(models.User).offset(skip).limit(limit).all()
 
 
-def create_user(db: Session, user: schemas.UserCreate):
+def create_user(db: Session, data: schemas.UserCreate):
     """Create a new user."""
-    password = user.user_password
+    password = data.user_password
     hashedpass = hash_password(password)
     db_user = models.User(
-        user_email=user.user_email,
-        user_first=user.user_first,
-        user_last=user.user_last,
-        user_hashed_password=str(hashedpass),
```

Comment:
If someone can explain why the cast to string was added, I'd be open to it - having this here, however, was failing tests.

Reply:
So Pycharm was yelling at me that it was expecting a string for that variable but it was byte type instead. So I cast it to string and recast to bytes further down in another function.